src/SourceBuild/content/eng/build.sourcebuild.targets

```diff
@@ -15,6 +15,17 @@
     <MSBuild Projects="$(RepoProjectsDir)$(RootRepo).proj" Targets="WritePrebuiltUsageData;ReportPrebuiltUsage" />
   </Target>
 
+  <!-- After building, copy SDK archive to assets root. -->
+  <Target Name="CopySdkArchive"
+          AfterTargets="Build">
+    <ItemGroup>
+      <_SdkTarball Include="$(ArtifactsAssetsDir)**/dotnet-sdk-*$(ArchiveExtension)" />
+    </ItemGroup>
+
+    <Copy SourceFiles="@(_SdkTarball)"
+          DestinationFolder="$(ArtifactsAssetsDir)" />
+  </Target>
+
```

Comment:
If we do this, we should do it for both build modes, not just source-build. I would move this into build.proj into the target that we already have that emits message to the console that points to the assets dir.

Reply:
Sounds good - will move it - thanks!

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86